packages/ckeditor5-media-embed/src/mediaembedediting.js

```diff
@@ -268,14 +269,10 @@ export default class MediaEmbedEditing extends Plugin {
 								contentId = callId;
 							}
 						} else {
-							if ( category === 'learning' ) {
-								contentType = 'learningpath';
-							} else if ( category === 'hubs' ) {
-								contentType = 'hub';
-							} else if ( category === 'feed/opportunities' ) {
+							if ( category === 'opportunities' ) {
```

Comment:
Ok maybe better to not have this special logic and make everything singular?

Reply:
Done